src/apt/apt/apt.go

```diff
@@ -104,6 +129,21 @@ func (a *Apt) AddRepos() error {
 			return err
 		}
 	}
+
+	if len(a.Priorities) > 0 {
+		prefFile, err := os.OpenFile(a.preferences, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0600)
```

Comment:
Use `0666` -- umask will adjust the permissions accordingly.

Reply:
Good call, will do!